doc/1.1.0/manual/intro.wiki

```diff
@@ -1,19 +1,91 @@
+<<head-css|
+div.screenshots p {
+  display:flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
+}
+div.screenshots img {
+  height: 200px;
+}
+>>
+
 =Introduction=
 
-Ocsigen Start is the fastest way to get started with Ocsigen. It
-consists of a library and an associated application template that
-combined implement standard functionality like users and
-notifications. The programmer can thus focus on the specificities of
-their application domain and quickly obtain a Minimum Viable Product
-(MVP).
+Ocsigen Start is an application template written with Eliom, Js_of_ocaml,
+Ocsigen Toolkit, etc.
+It contains many standard features like user management, notifications,
+and many code examples.
+
+Use it to learn Ocsigen or to quickly create your own Minimum Viable Product.
+It is probably the fastest way to get started with Ocsigen.
+
+The application is cross-platform: it works as a Web app or as a mobile app
+for iOS or Android (via [[https://cordova.apache.org/|Cordova]])..
+
+
+==Demo==
+
+If you want to test the demo application before installing Ocsigen Start,
+a working version is running [[wiki(81):demo/|here]].
+You can also download the mobile app
+[[https://play.google.com/store/apps/details?id=com.osdemo.mobile|on Google Play store]] (for Android)
+or download the
+[[wiki(81):files/osdemo.apk|APK for Android]]
+or the
+[[wiki(81):files/osdemo-ios.tgz|iOS version]] (to be installed using XCode).
+
+<<div class="screenshots"|
+[[files/screenshots/start1.png|
+{{files/screenshots/start1.png|Ocsigen Start}}]]
```

Comment:
You could also use the extension `a_img`  to abstract away the `files` directory, like so:
```suggestion
<<a_img src="screenshots/start1.png"|Ocsigen Start>>]]
```